### PR TITLE
Update gitignore and fix autoformat of calibration

### PR DESCRIPTION
**Pull request by nllong**
This PR does the following:

* Add a couple entries to the .gitignore
* The `calibration.py` file had a couple `10. * xyz` lines. Autopep converts this to `1 0. *`, which causes issues. This change does not impact the code, just updates the style.

### CODE REVIEW

File: testcases/bestest_hydronic_heat_pump/models/Resources/calibration/calibration.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def write_record_scroll(author, manufacturer, model, CoolingMode,
     if Q_nominal < 10.0e3:
         Q_str = (str(int(Q_nominal/1.0e3)) +
                  '_' +
-                 str(int(round(10.*(Q_nominal-np.floor(Q_nominal))/1.0e3))))
+                 str(int(round(10*(Q_nominal-np.floor(Q_nominal))/1.0e3))))
```
Comment (nllong) [inline testcases/bestest_hydronic_heat_pump/models/Resources/calibration/calibration.py:134]: this is one of the changes. The other items are just stripping whitespace.

File: testcases/bestest_hydronic_heat_pump/models/Resources/calibration/calibration.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def write_record_scroll(author, manufacturer, model, CoolingMode,
                      model,
                      Q_str + 'kW',
                      str(int(COP_nominal)),
-                     str(int(round(100.*(COP_nominal
+                     str(int(round(100*(COP_nominal
```
Comment (nllong) [inline testcases/bestest_hydronic_heat_pump/models/Resources/calibration/calibration.py:144]: This is the other change.

**Comment by dhblum** [comment]
Thanks for the PR.  gitignore and most formatting changes are fine by me.  @JavierArroyoBastida Can you just double check if the changes @nllong points out are of any consequence to ``calibration.py``?  In general, this python script is very specific to the development of the bestest hydronic with heat pump test case and is not core to BOPTEST.  It is provided so it is clear where some of the heat pump performance parameters came from for that test case.

**Comment by JavierArroyoBastida** [comment]
I've checked and these changes do not affect results so they can be safely implemented.

**Comment by dhblum** [comment]
Great, thanks @JavierArroyoBastida!

**Comment by nllong** [comment]
Thanks everyone!

**Pull request merged**